### ISSUE TITLE
Any idea to use this style

### ISSUE THREAD

**Issue by 2803media** [tilery/pianoforte#51]
Hi I have an instance with a tiles server based on the switch2osm tutorial and I wonder if I can use pianoforte style with this installation (build with osm2pgsql).

Thanks for your inputs, I really want to use this nice stylesheet!

Henri

**Comment by yohanboniface** [comment]
Nope, sorry, data needs to be imported with imposm, see README for installation instructions.
